tools/ci_connector_ops/ci_connector_ops/pipelines/README.md

```diff
@@ -125,14 +125,19 @@ The new metadata service also uses dagger for its reproducible CI pipeline.
 
 To see all available commands run
 ```bash
-airbyte-ci metadata-service --help
+airbyte-ci metadata --help
 ```
 
 For example to run the unit tests for the metadata service library you can run
 ```bash
 airbyte-ci metadata test lib
 ```
 
+For example to upload all the connector metadata files to GCS:
+```bash
+# You should set a valid JSON service account in the GCS_CREDENTIALS env var before uploading the files.
+airbyte-ci metadata upload <target-bucket> --all
```

Comment:
Im thinking we may want the bucket set as an env variable.

Reasoning is when we want to "go live" we could just change the github actions env var for GCS_CREDENTIALS and METADATA_SERVICE_BUCKET without any code change.

This value could default to our dev bucket.

What do you think?

Reply:
I don't think I get your suggestion. Even if we used an env var to set the bucket we'd need to switch the env var value in the GHA workflow. Maybe you suggest to use a GitHub secrets that we could change without code change. I still find it useful for discoverability to explicitly declare the bucket name in the GHA workflow and would prefer to not use a secret for it.

Reply:
Ah sorry, I had meant secret. But youve convinced me. Lets leave it searchable!